lib/fluent/plugin/parser_json.rb

```diff
@@ -70,16 +70,33 @@ def configure_json_parser(name)
       end
 
       def parse(text)
-        record = @load_proc.call(text)
-        time = parse_time(record)
-        if @execute_convert_values
```

Comment:
Note: No need to check `@execute_convert_values` here because it's also checked in `convert_values`.